### PR TITLE
Move setSenderName to the Message\SmsMessageInterface

### PR DESCRIPTION
**Pull request by almaudoh**
https://www.drupal.org/node/2842230

### CODE REVIEW

File: src/Message/SmsMessage.php [hunk 1]
```diff
@@ -17,6 +17,13 @@ class SmsMessage implements SmsMessageInterface {
   protected $uuid;
 
   /**
+   * The sender's name.
+   *
+   * @var string
```
Comment (dpi) [inline src/Message/SmsMessage.php]: |NULL

File: src/Message/SmsMessageInterface.php [hunk 2]
```diff
@@ -164,6 +164,25 @@ public function getResult();
...
+   *
+   * @return string|NULL
+   *   The name of the sender, or NULL if none is defined.
+   */
+  public function getSender();
+
+  /**
+   * Set the name of the sender of this SMS message.
+   *
+   * @param string $sender
```
Comment (dpi) [inline src/Message/SmsMessageInterface.php]: Although this wasn't there before; type should be `string|NULL`

**Review by dpi — COMMENTED** [review]
Address type issues then gtg

File: src/Message/SmsMessage.php (hunk 1 above)
Comment (almaudoh) [reply src/Message/SmsMessage.php]: Will fix. This was just a cut and paste of existing code. As an aside, per Drupal code standards, null in docs is not capitalized. But seeing as that's what we have in many other places, we can do that in a separate issue.

**Pull request closed without merging**